packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -148,6 +148,18 @@ describe("treeApi", () => {
 			const root = getView(config).root;
 			assert.equal(Tree.shortId(root), undefined);
 		});
+		it("returns the uncompressed identifier value when the provided identifier is an unknown stable id by the idCompressor.", () => {
+			const schemaWithIdentifier = schema.object("parent", {
+				identifier: schema.identifier,
+			});
+			const config = new TreeConfiguration(schemaWithIdentifier, () => ({
+				identifier: "invalidUUID",
+			}));
+
+			const root = getView(config).root;
+
+			assert.equal(Tree.shortId(root), "invalidUUID");
+		});
```

Comment:
I would recommend adding one more test for an identifier which is a valid UUID, but not a compressible UUID (i.e. not a stable key created by the compressor - just some UUID that passes `isStableNodeKey`)